internal/dev_server/ui/src/Flag.tsx

```diff
@@ -45,12 +47,18 @@ const VariationValues = ({
   switch (typeof flagValue) {
     case 'boolean':
       return (
-        <Switch
-          isSelected={currentValue}
-          onChange={(newValue) => {
-            updateOverride(flagKey, newValue);
-          }}
-        />
+        <div className="animated-switch-container">
```

Comment:
The styles broke for our launchpad component.  I want to get this fix out before debugging that, so for now I created a new style that styles correctly.

We were having issues with the styling regarding rems and correctly spacing the switch anyway, so this improves overall looks too at the cost of custom css.